pkg/sql/schemachanger/scpb/BUILD.bazel

```diff
@@ -17,6 +17,7 @@ go_library(
     deps = [
         "//pkg/sql/catalog/catpb",  # keep
         "//pkg/sql/sem/catid",  # keep
+        "//pkg/sql/sem/tree",  # keep
```

Comment:
No, please don't import `tree`! In fact, the enums you need are already in `catpb`, use those instead.